modal.css

```diff
@@ -0,0 +1,158 @@
+.subtittle {
+  margin-right: 2%;
+  float: left;
+}
+
+h2.popUp {
+  margin-top: 10%;
+  margin-bottom: 10%;
+  width: 50;
+}
+
+.popupContainer {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 95%;
+  z-index: 2;
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+  background: #fff;
+  justify-content: start;
+  margin-right: 3%;
+  margin-left: 3%;
+  margin-top: 4%;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  display: flex;
+  flex-direction: column;
+}
+
+.projectPopup {
+  display: grid;
+  height: 100%;
+  position: fixed;
+  padding: 0;
+  margin: 0;
+  background-color: #020a58;
+}
+
+.uil-multiply {
+  float: right;
+}
+
+.popupLinkContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  justify-content: start;
+  align-items: start;
+  width: 100%;
+}
+
+.popupLinkContainer a {
+  margin: 4%;
+  color: #396df2;
+  padding: 0.5rem 0.5rem;
+  text-align: center;
+  cursor: pointer;
+  border: 1px solid #396df2;
+  border-radius: 9px;
+}
+
+.popupLinkContainer a:focus {
+  background-color: #020a58;
+}
+
+.popupLinkContainer a:hover {
+  background-color: #020a58;
+  color: white;
+  transition: 0.2s;
+}
```

Comment:
- [ ] I like that you're adding your own taste to the design here but you'll have to make sure you follow the given design at least at this stage as one of the objectives of this module is to give you the skill to implement any given design to near perfection 🎯
- Here's the Figma design for the hover state of the button:
  ![firefox_7KIb3eS8J1](https://user-images.githubusercontent.com/104968236/233395759-87b34909-50e7-4345-8ca7-a273e978acd0.png)